xtc1to2/read_img.py

```diff
@@ -45,6 +45,9 @@ def __init__(self, exp, run, mode, detector_name):
 
         # Set up detector
         self.detector = psana.Detector(detector_name)
+        # set flag (for Chuck)
+        self.detector.do_reshape_2d_to_3d(flag=True) 
```

Comment:
I guess I don't understand this but I think @chuckie82 and @smarkesini know where this function needs to be added.